ansible/development.yml

```diff
@@ -0,0 +1,10 @@
+---
+- hosts: all
+  sudo: yes
+  roles:
+    - common
+    - rsyslog
+    - nginx
+    - mariadb
```

Comment:
Not sure if we need to save anything to DB (given the current knowns, I assume we don't); we can add it when we need to.